AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Campaign/Details.cshtml

```diff
@@ -11,7 +13,31 @@
 
 <div class="row">
     <div id="col-main">
-        <div class="col-md-9"><h2>@Model.Name <a asp-controller="Campaign" asp-action="Edit" asp-route-id="@Model.Id" class="btn btn-default" title="Edit"><i class="fa fa-edit"></i></a> <a asp-controller="Campaign" asp-action="Delete" asp-route-area="Admin" asp-route-id="@Model.Id" class="btn btn-default"><i class="glyphicon glyphicon-trash"></i></a></h2></div>
+        <div class="col-md-9">
+            <h2 class="inline-form">
+                @Model.Name
+            </h2>
+            <a asp-controller="Campaign" asp-action="Edit" asp-route-id="@Model.Id" class="btn btn-default inline-form" title="Edit"><i class="fa fa-edit"></i></a>
+            <a asp-controller="Campaign" asp-action="Delete" asp-route-area="Admin" asp-route-id="@Model.Id" class="btn btn-default inline-form"><i class="glyphicon glyphicon-trash"></i></a>
+            @if (User.IsUserType(UserType.SiteAdmin))
+            {
+                <form asp-controller="Campaign" asp-route-area="Admin" asp-action="LockUnlock" class="inline-form">

```

Comment:
On this one it's not presenting any other page or input via a http get so I felt it best practice to make this require a http post to the action via a form. If we make it just a hyperlink we potentially should add a page to confirm that we want to lock or unlock the campaign which felt unnecessary.